tests/config/lightning_api.settings.yml

```diff
@@ -1,2 +1,3 @@
 entity_json: true
+bundle_docs: false
```

Comment:
Whoops. We should have had this in a previous PR, but I guess we forgot. No harm done.